pkg/config/config_template.yaml

```diff
@@ -2834,6 +2834,42 @@ api_key:
   #   - '<COMMUNITY_1>'
   #   - '<COMMUNITY_2>'
 
+  ## @param users - list of custom objects - optional
+  ## List of SNMPv3 users that can be used to listen for traps.
+  ## NOTE: Currently the Datadog Agent only supports having a
+  ## single user in this list.
```

Comment:
`@param` line says this is a list of objects, but the examples below show a single nested object; these need to be aligned (either a list or an object). Unless there is an immediate plan to support multiple users, may I propose we call this parameter `user` and make it a single object? A `users` list then can be added later in a backwards compatible way. Then this note becomes unnecessary.

Reply:
The example below is wrong indeed and should start with an `-` to match the implementation.
There is a plan to work on supporting multiple users soon but it requires additional work in the `gosnmp` library first.

The idea here was to keep as much backwards compatibility as possible, by having `user` and later `users` the code would need to continue supporting both for backwards compatibility which we'd rather not do if possible